### DISCUSSION TITLE
View/edit/export a "Data Asset"

### DISCUSSION THREAD

**Discussion by kripper** [gildor2/UEViewer#282] [Ideas]
Hi,

I want to view/edit a "Data Asset" (a Pavlov's "definition.uasset" file).
Any support for this?

I changed a property using the editor and found out there are two changes in the binary file: The property value (easy to decode) and another chunk which I guess could be a checksum.

I would like to change a property in a compiled uasset file, or at least, export its values so I can re-generate it from scratch with the same properties.

**Comment by gildor2** [comment]
Hi. There never was support for editing anything with my tools.

**Comment by kripper** [comment]
And viewing or exporting data asset properties?

**Reply by gildor2** [reply]
No. Only models resources are supported.
